handlers/events.go

```diff
@@ -0,0 +1,58 @@
+package handlers
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/hashicorp/serf/serf"
+	"github.com/julienschmidt/httprouter"
+	"github.com/livepeer/catalyst-api/cluster"
+	"github.com/livepeer/catalyst-api/errors"
+	"github.com/xeipuuv/gojsonschema"
+	"io"
+	"net/http"
+)
+
+type EventsHandlersCollection struct {
+	Cluster cluster.Cluster
+}
+
+func (d *EventsHandlersCollection) Events() httprouter.Handle {
```

Comment:
Is there any authentication on this API? Not a critical issue but might wanna consider adding something, if nothing yet (like the shared secret from task-runner <-> catalyst)

Reply:
For now not. But I think we don't need it right now. YAGNI..hehe. We don't event know if this endpoint will need an authentication. If we move back to the idea of EIP-712 signed events, then this endpoint won't even require any auth.